tests/entrypoint/database/test_add.py

```diff
@@ -36,34 +35,60 @@ def test_mysql_not_implemented():
         main(["add", "a", "b", "--db-type=mysql"])
 
 
-def test_sqlite_cord19(engine_sqlite, tmpdir):
-    input_folder = pathlib.Path(str(tmpdir))
+def test_sqlite_cord19(engine_sqlite, tmp_path):
+    path_to_pkl = tmp_path / "pkl_files"
+    path_to_pkl.mkdir()
     n_files = 3
 
-    input_paths = [input_folder / f"{i}.pkl" for i in range(n_files)]
-
-    for i, input_path in enumerate(input_paths):
-        article = Article(
+    pkl_files = [path_to_pkl / f"{i}.pkl" for i in range(n_files)]
+    articles = [
+        Article(
             title=f"title_{i}",
             authors=[f"author_{i}"],
             abstract=f"abstract_{i}",
             section_paragraphs=[("Conclusion", f"conclusion_{i}")],
         )
-        with open(input_path, "wb") as f:
+        for i in range(n_files)
+    ]
+    for article, pkl_file in zip(articles, pkl_files):
+        with pkl_file.open("wb") as f:
             pickle.dump(article, f)
 
+    query = "SELECT COUNT(*) FROM articles"
+
+    # Test adding single article
+    for pkl_file in pkl_files:
         args_and_opts = [
             "add",
             engine_sqlite.url.database,
-            str(input_path),
+            str(pkl_file),
+            "--db-type=sqlite",
+        ]
+        main(args_and_opts)
+    (n_rows,) = engine_sqlite.execute(query).fetchone()
+    assert n_rows == n_files
+
+    engine_sqlite.execute("DELETE FROM articles WHERE True")
+    (n_rows,) = engine_sqlite.execute(query).fetchone()
+    assert n_rows == 0
```

Comment:
What does this do exactly? I'm not familar with `WHERE True`.

Reply:
It just drops all rows in the table.

`WHERE True` is not needed per se, I just put it there to make explicit my intent of deleting _all_ rows in the table. See e.g. StackOverflow [here](https://stackoverflow.com/a/37849060).

For some reason, `DELETE FROM articles` sounds a bit awkward to me if there is no `WHERE ...` clause following it 😁 
But if you find it confusing I can change it!

Reply:
Changed in 50257b0.

Reply:
I see, thanks for the explanation! :)